### DISCUSSION TITLE
/baltop not updating my amount of money

### DISCUSSION THREAD

**Discussion by TSGPL** [EssentialsX/Essentials#4064] [Q&A] [answered]
So i have an issue where i (TSGPL) show up in /baltop with 0 dollars tho i have 600k+ (see screenshots)
i already tried looking up the issue and tried to fix it and i removed the baltop exclude permission on the Owner and Admin rank and myself as well.
i also removed the * permission from myself and removed my op and the issue still persists

![baltop not working](https://user-images.githubusercontent.com/73583048/111466810-078a2b00-8724-11eb-8057-bf4f3743012c.png)

**Comment by JRoy** [comment, marked as answer]
Cannot reproduce on latest dev build with new Baltop API
